apps/src/applab/applab.js

```diff
@@ -647,9 +641,9 @@ Applab.init = function (config) {
   }
 
   // If we are in mobile sharing mode, allow the viewport to handle scaling
-  // and override our default width target in vizAppWidth with the actual width
+  // and override our default width target in VIZ_APP_WIDTH with the actual width
   if (dom.isMobile() && config.hideSource) {
-    vizAppWidth = Applab.appWidth;
+    VIZ_APP_WIDTH = Applab.appWidth;
```

Comment:
You are right, this should not be all caps. Somehow didn't notice that this was changing